sunxi-boot0.c

```diff
@@ -18,7 +18,7 @@
 #define BOOT0_CHECKSUM	3
 #define BOOT0_LENGTH	4
 #define BOOT0_DRAM_OFS	14
-#define OFS(x) ((x) + BOOT0_DRAM_OFS)
+#define OFS(x) ((x) + 1)
```

Comment:
Please add a comment stating the purpose of "+ 1" here. I guess it's to differentiate it from the special zero case?